src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@carlosmiranda I think, my diagnostic was again wrong. RqLive separate the header with body with an empty line "". So the code in Rqlive is fine.